services/package-manager/src/test/java/org/openthinclient/pkgmgr/it/PackageInstallTest.java

```diff
@@ -56,7 +58,44 @@ public void testInstallSinglePackage() throws Exception {
     assertTrue(fooPackage.isPresent());
     assertTrue(packageManager.install(Collections.singletonList(fooPackage.get())));
 
-    // verify the existence files from the package.
+    File fooXml = new File(configuration.getInstallDir().getPath() + File.separator + "schema" + File.separator + "application" + File.separator + "foo.xml");
+    File fooSample = new File(configuration.getInstallDir().getPath() + File.separator + "schema" + File.separator + "application" + File.separator + "foo-tiny.xml.sample");
+    File fooSfs = new File(configuration.getInstallDir().getPath() + File.separator + "sfs" + File.separator + "package" + File.separator + "foo.sfs");
+
+    assertTrue("foo.xml not installed",fooXml.exists());
+    assertTrue("foo-tiny.xml.sample not installed",fooSample.exists());
+    assertTrue("foo.sfs not installed",fooSfs.exists());
+    
+    assertEquals(0,configuration.getTestinstallDir().listFiles().length);
+  }
+  
+  @Test
+  public void testInstallSinglePackageDependingOther() throws Exception {
+    final DPKGPackageManager packageManager = preparePackageManager();
+
+    final Optional<org.openthinclient.util.dpkg.Package> bar2Package = packageManager.getInstallablePackages()
+            .stream()
+            .filter(pkg -> pkg.getName().equals("bar2"))
+            .findFirst();
+
+    assertTrue(bar2Package.isPresent());
+    assertTrue(packageManager.install(Collections.singletonList(bar2Package.get())));
+
+    File fooXml = new File(configuration.getInstallDir().getPath() + File.separator + "schema" + File.separator + "application" + File.separator + "foo.xml");
+    File fooSample = new File(configuration.getInstallDir().getPath() + File.separator + "schema" + File.separator + "application" + File.separator + "foo-tiny.xml.sample");
+    File fooSfs = new File(configuration.getInstallDir().getPath() + File.separator + "sfs" + File.separator + "package" + File.separator + "foo.sfs");
+    File bar2Xml = new File(configuration.getInstallDir().getPath() + File.separator + "schema" + File.separator + "application" + File.separator + "bar2.xml");
+    File bar2Sample = new File(configuration.getInstallDir().getPath() + File.separator + "schema" + File.separator + "application" + File.separator + "bar2-tiny.xml.sample");
+    File bar2Sfs = new File(configuration.getInstallDir().getPath() + File.separator + "sfs" + File.separator + "package" + File.separator + "bar2.sfs");
+
+    assertTrue("foo.xml not installed",fooXml.exists());
```

Comment:
In conjunction with the comment above, it would be nice to introduce custom assertion Methods which will require less maintenance. 

````
public void assertFileExists(Path path) {
  assertTrue(path.getFileName() + " does not exist", Files.exists(path));
  assertTrue(path.getFileName() + " is not a regular file", Files.isRegularFile(path));
}
```